packages/mobile/src/pincode/PincodeSet.test.tsx

```diff
@@ -134,4 +136,36 @@ describe('Pincode', () => {
     await flushMicrotasksQueue()
     expect(getByText('pincodeSet.pinsDontMatch')).toBeDefined()
   })
+
+  it('navigates to the Settings screen after successfully changing PIN', async () => {
+    const { getByTestId, rerender } = render(
+      <Provider store={mockStore}>
+        <PincodeSet {...mockScreenProps} />
```

Comment:
`mockScreenProps` has `isVerifying` as true, so it'll start with that. And then we're rerendering with `isVerifying` as `false`? Did you run this test and does it pass? Things seem to be in the incorrect order.

Reply:
after the user has entered their new pin, `isVerifying` should be true, so yeah i think it should be true

Reply:
I think what you want here is for `mockScreenProps` to have to have `changePin: true` so that the `PincodeSet` screen knows that we are navigating here from the `Settings` screen. `isVerifying` should only be set on the 2nd render because that's how the screen knows that this is the 2nd time the user is entering their pin and it should be checking if it's the same as the first entry